flang/lib/Lower/OpenMP.cpp

```diff
@@ -110,9 +110,11 @@ genOMP(Fortran::lower::AbstractConverter &converter,
       std::get<Fortran::parser::OmpBeginBlockDirective>(blockConstruct.t);
   const auto &parallelDirective =
```

Comment:
Can you rename this? 
```suggestion
  const auto &blockDirective =
```

Reply:
Sure, Thanks!